src/seq/mod.rs

```diff
@@ -674,6 +674,7 @@ pub struct SliceChooseIter<'a, S: ?Sized + 'a, T: 'a> {
 }
 
 #[cfg(feature = "alloc")]
+#[cfg_attr(doc_cfg, doc(cfg(feature = "alloc")))]
 impl<'a, S: Index<usize, Output = T> + ?Sized + 'a, T: 'a> Iterator for SliceChooseIter<'a, S, T> {
```

Comment:
This doesn't show up anywhere because `SliceChooseIter` is already `cfg(feature = "alloc")` and `Iterator` is not a local trait.

Reply:
Is it worth leaving it here in case rustdoc picks it up later?